relayer/middleware/wallet/walletToolBox.ts

```diff
@@ -18,10 +19,7 @@ export function createWalletToolbox(
   }
   switch (chainId) {
     case wh.CHAIN_ID_SOLANA:
-      return createSolanaWalletToolBox(
-        providers,
-        new Uint8Array(JSON.parse(privateKey))
-      );
+      return createSolanaWalletToolBox(providers, bs58.decode(privateKey));
```

Comment:
Yeah, this is a good point that I forgot to point out in the description, my main motivation for this was to avoid having to `JSON.parse` environment variables so that we could read the key straight from the `env`. Supporting both is a good idea imho